index.js

```diff
@@ -1,17 +1,59 @@
 class SortedList {
-  constructor() {}
 
-  add(item) {}
 
-  get(pos) {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  max() {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length++;
+  }
 
-  min() {}
+  get(pos) {
+      pos = this.items[pos];
+      if (!pos) {
+        throw new Error('OutOfBounds');
+      } else {
+        return pos;
+      }
```

Comment:
es un poco lioso, mejor así `
      if (!this.items[pos]) {
        throw new Error('OutOfBounds');
      } else {
        return this.items[pos];
      }`